### PR TITLE
feat: add more message translations

### PR DESCRIPTION
**Pull request by mirsadikov**
- refactored exception filter to translate errors in one place
- removed unncessary success messages like "success fetching list"
- added translation for success messages 

**Most changes are for syntax and naming consistency accross the project**

### CODE REVIEW

**Review by zilola08 — APPROVED** [review]
Well done! 👍

**Review by Sinumi — APPROVED** [review]
approve

File: src/filters/global.filter.ts [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
     try {
+      const knownException = exception instanceof HttpException;
       const request = ctx.getRequest<Request>();
-      const status = exception.getStatus();
 
-      const errorObj = exception.getResponse();
+      const status = knownException ? exception.getStatus() : 500;
+      const errorObj = knownException ? exception.getResponse() : exception;
+      const key = knownException ? errorObj?.key || errorObj?.message : ErrorsTypes.DEFAULT;
+      const message = knownException ? translateErrorMessage(this.i18n, key) : exception.message;
```
Comment (gitkoshelew) [inline src/filters/global.filter.ts:31]: looks like you repeat yourself
make default assignment first
let logString =  `method: ${request.method}, to: ${request.originalUrl}, status: 500, errorMessage: ${exception.getResponse().key}, cause: ${errorObj.message}`

if (exception instanceof HttpException) {
   logString = ...
}


of make particular assignment 

      let status =  500 (should be enum or const here)
      let errorObj = exception;
      let key = ErrorsTypes.DEFAULT;
      let message =  exception.message;

if ( exception instanceof HttpException) {
status = ....

File: src/filters/global.filter.ts [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
+    private readonly i18n: I18nService
+  ) {}
 
   catch(exception: GlobalException | any, host: ArgumentsHost) {
     const ctx = host.switchToHttp();
     const response = ctx.getResponse<Response>();
+    const defaultErrorMessage = translateErrorMessage(this.i18n, ErrorsTypes.DEFAULT);
+    const defaultResponse = {
+      success: false,
+      error: { statusCode: 500, message: defaultErrorMessage },
```
Comment (gitkoshelew) [inline src/filters/global.filter.ts:21]: create enum or get 500 from nest constants

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

File: src/filters/global.filter.ts (hunk 1 above)
Comment (mirsadikov) [reply src/filters/global.filter.ts]: i changed, is it ok now?

**Review by gitkoshelew — APPROVED** [review]

**Pull request merged**